southworks/botframework-solutions
# [Botskills] Change from warning to informative message

## Purpose
_What is the context of this pull request? Why is it being done?_
The user may be confused by the warning message when there are no authentication connections to configure during the Skill connection.

## Changes
_Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)_
Replace the warning if there are no authentication connections to an informative message. The initial condition was also modified to contemplate the cases of an empty array.

![image](https://user-images.githubusercontent.com/40918752/65537573-d8eb1580-dedb-11e9-88ab-b6b9b3865e5e.png)

## Tests
_Is this covered by existing tests or new ones? If no, why not?_
`authentication.test.js` covers the changes that are being added.
### Testing Steps
1. Deploy a `Virtual Assistant` and `Skill`.
2. Go to `.\templates\Virtual-Assistant-Template\typescript\samples\sample-assistant`.
3. Open a terminal in that location and establish a connection between the `Virtual Assistant` and `Skill`, make sure to make a connection without authentication, by having an empty array in the `authenticationConnections` file `manifestTemplate.json`.
4. Check that the image matches.

## Feature Plan
_Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues._
\-

### Checklist
#### General

#### Bots

#### Deployment Scripts
